projects/js-packages/publicize-components/src/components/connection-management/connection-info.tsx

```diff
@@ -11,10 +20,44 @@ type ConnectionInfoProps = ConnectionStatusProps;
  * @returns {import('react').ReactNode} - React element
  */
 export function ConnectionInfo( { connection, onReconnect }: ConnectionInfoProps ) {
+	const [ isPanelOpen, togglePanel ] = useReducer( state => ! state, false );
+
 	return (
-		<div>
-			<ConnectionName connection={ connection } />
-			<ConnectionStatus connection={ connection } onReconnect={ onReconnect } />
-		</div>
+		<>
+			<div className={ styles[ 'connection-item' ] }>
+				<ConnectionIcon
+					serviceName={ connection.service_name }
+					label={ connection.display_name }
+					profilePicture={ connection.profile_picture }
+				/>
+				<div className={ styles[ 'connection-item-name' ] }>
+					<ConnectionName connection={ connection } />
+				</div>
+				<ConnectionStatus connection={ connection } onReconnect={ onReconnect } />
+				<Button
+					size={ 'small' }
+					className={ styles[ 'learn-more' ] }
+					variant="tertiary"
+					onClick={ togglePanel }
+					aria-label={ __( 'Learn more', 'jetpack' ) }
+				>
+					{ <Icon className={ styles.chevron } icon={ isPanelOpen ? chevronUp : chevronDown } /> }
+				</Button>
+			</div>
+			<Panel className={ styles[ 'service-panel' ] }>
```

Comment:
Isn't this a `connection-panel` instead of a social `service-panel`?

Reply:
Good point, I grabbed this from your change, I'll updateit